From 031780293d2838da2643b46878061598ebb56822 Mon Sep 17 00:00:00 2001
From: Marcin Juszkiewicz <mjuszkiewicz@redhat.com>
Date: Thu, 8 Oct 2015 10:24:36 +0200
Subject: [PATCH] DO NOT hardcode x86-64 architecture.

doxygen built on most of architectures by pure luck and order of Qt5
qatomics_arch.h header where check for QT_ARCH_X86_64 was *after* most
of other architectures.

But not after AArch64 where it failed due to attempt of using x86-64
atomics code.

https://github.com/doxygen/doxygen/pull/402

Signed-off-by: Marcin Juszkiewicz <mjuszkiewicz@redhat.com>
---
 addon/doxywizard/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: doxygen-1.8.10/addon/doxywizard/CMakeLists.txt
===================================================================
--- doxygen-1.8.10.orig/addon/doxywizard/CMakeLists.txt
+++ doxygen-1.8.10/addon/doxywizard/CMakeLists.txt
@@ -10,7 +10,7 @@ include_directories(
 set(GENERATED_SRC_WIZARD ${GENERATED_SRC}/doxywizard)
 file(MAKE_DIRECTORY ${GENERATED_SRC_WIZARD})
 
-add_definitions(-DQT_ARCH_X86_64 -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_TO_ASCII -DUNICODE)
+add_definitions(-DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_TO_ASCII -DUNICODE)
 set(QT_USE_QTXML TRUE)
 find_package(Qt4 REQUIRED)
 include(${QT_USE_FILE})
